RELEASE_TEMPLATE.md

```diff
@@ -37,38 +37,52 @@ Times are relative to the start of the release.
-- [ ] Ensure that `deploy/docker/conf/docker-compose.yml` has been updated with the correct image versions.
-- [ ] Ensure that `deploy/aws/conf/cloudformation_template.json` has been updated with the correct image versions.
-- [ ] Ensure that `IamRoleInstance,IamRoleService,IamRoleTask,IamRoleLambda` in `deploy/aws/conf/cloudformation_template.json` have the correct role permissions.
-- [ ] Ensure that a $$blox_version$$ change log entry exists in `CHANGELOG.md`.
-- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/README.md` and `daemon-scheduler/README.md`.
-- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/VERSION` and `daemon-scheduler/VERSION`.
-- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/versioning/version.go` and `daemon-scheduler/versioning/version.go`.
-- [ ] Ensure that the technician has a GitHub account and permissions to merge into the Blox master branch.
-- [ ] Ensure that the technician has a Docker Hub account and permissions to push into the bloxoss repositories.
+- [ ] Ensure that `deploy/docker/conf/docker-compose.yml` has been updated with the correct image versions in the release-$$blox_version$$ branch.
+- [ ] Ensure that `deploy/aws/conf/cloudformation_template.json` has been updated with the correct image versions in the release-$$blox_version$$ branch.
+- [ ] Ensure that `IamRoleInstance,IamRoleService,IamRoleTask,IamRoleLambda` in `deploy/aws/conf/cloudformation_template.json` have the correct role permissions in the release-$$blox_version$$ branch.
+- [ ] Ensure that a $$blox_version$$ change log entry exists in `CHANGELOG.md` in the release-$$blox_version$$ branch.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/README.md` and `daemon-scheduler/README.md` in the release-$$blox_version$$ branch.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/VERSION` and `daemon-scheduler/VERSION` in the release-$$blox_version$$ branch.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/versioning/version.go` and `daemon-scheduler/versioning/version.go` in the release-$$blox_version$$ branch.
+- [ ] Ensure that the release-$$blox_version$$ branch contains the latest changes in the dev and master branch.
+- [ ] Ensure that the release-$$blox_version$$ branch commit hash is set to $$github_hash$$.
+- [ ] Ensure that the technician has a GitHub account and permissions to merge into the Blox dev and master branches.
+- [ ] Ensure that the technician has a Docker Hub account and permissions to push and remove images in the bloxoss repositories.
 - [ ] Ensure that the technician has logged into Docker Hub via `docker logout` and `docker login`.
+- [ ] Delete all bloxoss local images before building new ones.
+- [ ] Build the bloxoss/cluster-state-service:pre-$$blox_version$$ image and publish to Docker Hub.
```

Comment:
I would argue that it's better to test the image with the actual version tag and signature that you're going to release. Since a private registry (ECR) provides a way for doing that, I'd highly encourage doing that instead of `pre` tags. For example, let's say ECR was the way that we shared images with everyone. The process then would be to not have the "accessible to everyone" flag for the image, test it and them make it public as part of the release process.